src/MarkdownEditor2022.csproj

```diff
@@ -1,157 +1,95 @@
-    <Compile Include="Language\MarkdownEditor.cs" />
-    <Compile Include="Language\MarkdownClassificationTypes.cs" />
-    <Compile Include="Language\TokenTagger.cs" />
-    <Compile Include="Margin\Browser.cs" />
-    <Compile Include="Margin\BrowserMargin.cs" />
-    <Compile Include="Margin\BrowserMarginProvider.cs" />
-    <Compile Include="Options\AdvancedOptions.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Commands\EnablePreviewSyncCommand.cs" />
-    <Compile Include="MarkdownEditor2022Package.cs" />
-    <Compile Include="source.extension.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>source.extension.vsixmanifest</DependentUpon>
-    </Compile>
-    <Compile Include="VSCommandTable.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>VSCommandTable.vsct</DependentUpon>
-    </Compile>
+    <AdditionalThisAssemblyFields Include="AssemblyDescription" String="$(Description)" />
   </ItemGroup>
+
   <ItemGroup>
-    <Content Include="Margin\highlight-dark.css">
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="Margin\md-template.html">
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="..\LICENSE">
-      <Link>Resources\LICENSE</Link>
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <None Include="source.extension.vsixmanifest">
-      <SubType>Designer</SubType>
-      <Generator>VsixManifestGenerator</Generator>
-      <LastGenOutput>source.extension.cs</LastGenOutput>
-    </None>
-    <Content Include="Margin\highlight.css">
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="Margin\prism-dark.css">
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="Margin\prism.css">
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="Margin\prism.js">
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
-    <Content Include="Resources\Icon.png">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-      <IncludeInVSIX>true</IncludeInVSIX>
-    </Content>
+    <Content Include="..\LICENSE" Link="Resources\LICENSE" IncludeInVSIX="true" />
+    <Content Include="Margin\highlight-dark.css" IncludeInVSIX="true" />
+    <Content Include="Margin\highlight.css" IncludeInVSIX="true" />
+    <Content Include="Margin\md-template.html" IncludeInVSIX="true" />
+    <Content Include="Margin\prism-dark.css" IncludeInVSIX="true" />
+    <Content Include="Margin\prism.css" IncludeInVSIX="true" />
+    <Content Include="Margin\prism.js" IncludeInVSIX="true" />
+    <Content Include="Resources\Icon.png" CopyToOutputDirectory="PreserveNewest" IncludeInVSIX="true" />
   </ItemGroup>
+
   <ItemGroup>
-    <VSCTCompile Include="VSCommandTable.vsct">
-      <ResourceName>Menus.ctmenu</ResourceName>
-      <Generator>VsctGenerator</Generator>
```

Comment:
📝 This might be valuable, but we should create a Roslyn source generator (will not be difficult) to generate this file during the build directly from the VSCT

Reply:
It needs the Generator to produce a .cs file used by the extension. We could switch to source generators once they exist for the .vsct and .vsixmanifest files

Reply:
➡️ The source generator is now implemented and included in this PR

Reply:
➡️ This change is moved to #10